Source/Scene/Cesium3DTileset.js

```diff
@@ -497,6 +522,63 @@ define([
         }).otherwise(function(error) {
             that._readyPromise.reject(error);
         });
+
+        /**
+         * Determines if level-of-detail skipping optimization should be used.
+         *
+         * @type {Boolean}
+         * @default true
+         */
+        this.skipLODs = defaultValue(options.skipLODs, true);
+
+        /**
+         * This is a multiplier defining the minumum screen space error to skip.
+         * If current tile has screen space error of 100, no tiles will be loaded unless they
+         * are leaves or have a screen space error <= 100 / skipSSEFactor.
+         *
+         * Only used when tileset.skipLODs === true.
+         *
+         * @type {Number}
+         * @default 10
+         */
+        this._skipSSEFactor = defaultValue(options.skipSSEFactor, 10);
```

Comment:
The `_` properties should not be documented here; instead, they should have readonly getter functions below.